tests/sentry/issues/test_ingest.py

```diff
@@ -452,13 +452,14 @@ def test_new_group_with_priority(self) -> None:
 
 class CreateIssueKwargsTest(OccurrenceTestMixin, TestCase):
     def test(self) -> None:
-        occurrence = self.build_occurrence()
+        occurrence = self.build_occurrence(culprit="abcde" * 100)
         event = self.store_event(data={}, project_id=self.project.id)
         assert _create_issue_kwargs(occurrence, event, None) == {
             "platform": event.platform,
             "message": event.search_message,
             "level": LOG_LEVELS_MAP.get(occurrence.level),
-            "culprit": occurrence.culprit,
+            # Should truncate the culprit to max allowable length
+            "culprit": f"{occurrence.culprit[:MAX_CULPRIT_LENGTH-3]}...",
```

Comment:
why -3?

Reply:
Accounts for the ellipsis which get appended by the `truncatechars` utility. I could probably just use it in the test but this felt more transparent.